docs/scanners.md

```diff
@@ -177,6 +177,7 @@ Scanners:
 
 90001   Insecure JSF ViewState
 90002   Java Serialized Object
+90003   SRI integrity missing attribute
```

Comment:
The `missing` should be at the end?

Reply:
Good catch. Also the `I` in the accronym is technically `Integrity` which is redundant to include twice so how about either:
`SRI Attribute Missing`
or
`Subresource Integrity Attribute Missing`

Reply:
Last one looks good to me.